login-workflow/package.json

```diff
@@ -75,13 +76,13 @@
         "@babel/preset-typescript": "^7.14.5",
         "@brightlayer-ui/eslint-config": "^3.0.0",
         "@brightlayer-ui/prettier-config": "^1.0.3",
-        "@brightlayer-ui/react-components": "^6.1.2-alpha.0",
```

Comment:
Please update @brightlayer-ui/react-themes to 8.0.0-alpha.0

Reply:
changes done